src/VS.Web.CG.Mvc/Identity/identitygeneratorfilesconfig.json

```diff
@@ -5,7 +5,9 @@
         "Name": "_LoginPartial",
         "SourcePath": "_LoginPartial.cshtml",
         "OutputPath": "Pages/Shared/_LoginPartial.cshtml",
-        "IsTemplate": true
+        "IsTemplate": true,
+        "AltPaths": [ "Pages/_LoginPartial.cshtml" ],
+        "ShouldOverWrite": "Never"
```

Comment:
What does the "ShouldOverwrite":"Never" meam?

Reply:
When identity scaffolding occurs, files marked with ShouldOverWrite = Never are checked for existence in the project being scaffolded. If the file exists, then scaffolding doesn't make a new one - doesn't overwrite the same-named file that the user has already created. 

If the project being scaffolded does not have this file, it can be created.  But it only gets created if the user has chosen that file to be created (or selected all files).